wb_vandalism/datasources/revision_metadata.py

```diff
@@ -0,0 +1,10 @@
+
+from revscoring.datasources import Datasource
+from revscoring.datasources.revision import metadata
+
+
+def process_comment(metadata):
```

Comment:
Why not just require metadata and access the comment that way?  Does having comment as its own datasource get us anything?